clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/Main.groovy

```diff
@@ -88,20 +86,12 @@ class Main extends SpringBootServletInitializer {
 
   @Bean
   @Primary
```

Comment:
Not a fan of `@Primary`, but it was here before https://github.com/spinnaker/clouddriver/pull/5976/files#diff-696765285cfe1add5c28a1a59f9d7ad2249e9bc9e9383104767c076af686bc64R90 added the artifact store stuff.

Reply:
Agreed! I think we can figure out a better pattern at some point, but clouddriver relies on it too heavily at the moment. So we can take some time to refactor this at some point